packages/components/inputs/localized-rich-text-input/src/localized-rich-text-input.stories.tsx

```diff
@@ -138,7 +138,6 @@ BasicExample.args = {
   showExpandIcon: false,
   selectedLanguage: 'en',
   hideLanguageExpansionControls: false,
-  isAutofocussed: false,
```

Comment:
(removed, cause the prop does not exist on the localized version of the rich-text-input)